src/tests/putaway/putawayMoreThan1Item.test.ts

```diff
@@ -0,0 +1,412 @@
+import AppConfig from '@/config/AppConfig';
+import { ShipmentType } from '@/constants/ShipmentType';
+import { expect, test } from '@/fixtures/fixtures';
+import { StockMovementResponse } from '@/types';
+import { getShipmentId, getShipmentItemId } from '@/utils/shipmentUtils';
+
+test.describe('Create putaway for more than 1 item, separate putaways', () => {
+  let STOCK_MOVEMENT: StockMovementResponse;
+
+  test.beforeEach(
+    async ({
+      supplierLocationService,
+      stockMovementService,
+      fifthProductService,
+      fourthProductService,
+      receivingService,
+    }) => {
+      const supplierLocation = await supplierLocationService.getLocation();
+      STOCK_MOVEMENT = await stockMovementService.createInbound({
+        originId: supplierLocation.id,
+      });
+
+      const product = await fifthProductService.getProduct();
+      const product2 = await fourthProductService.getProduct();
+
+      await stockMovementService.addItemsToInboundStockMovement(
+        STOCK_MOVEMENT.id,
+        [
+          { productId: product.id, quantity: 10 },
+          { productId: product2.id, quantity: 10 },
+        ]
+      );
+
+      await stockMovementService.sendInboundStockMovement(STOCK_MOVEMENT.id, {
+        shipmentType: ShipmentType.AIR,
+      });
+
+      const { data: stockMovement } =
+        await stockMovementService.getStockMovement(STOCK_MOVEMENT.id);
+      const shipmentId = getShipmentId(stockMovement);
+      const { data: receipt } = await receivingService.getReceipt(shipmentId);
+      const receivingBin =
+        AppConfig.instance.receivingBinPrefix + STOCK_MOVEMENT.identifier;
+
```

Comment:
I think I commented about it in the second PR - it will be good to have it as a separated getter